.builds/freebsd.yml

```diff
@@ -21,14 +21,15 @@ tasks:
   - test: |
       cd alacritty
       cargo test
+  - oldstable: |
+      cd alacritty
+      rustup toolchain install --profile minimal 1.45.2
```

Comment:
Need to fixup your rebase, since this commit hasn't been merged yet.

Reply:
I've intentionally rebased on top of this since without it CI cannot succeed. I'll fix the rebase issues once the other PR is merged.